bliss/models/location_encoder.py

```diff
@@ -374,16 +335,21 @@ def _encode_for_n_sources(
 
         # finally, we slice pps5 by parameter group because these groups are treated differently,
         # subsequently
-        split_sizes = [v["dim"] for v in self.variational_params.values()]
-        var_params_split = torch.split(pps5, split_sizes, 3)
-        names = self.variational_params.keys()
-        var_params_for_n_sources = dict(zip(names, var_params_split))
+        split_sizes = [v["dim"] for v in self.dist_params.values()]
+        dist_params_split = torch.split(pps5, split_sizes, 3)
+        names = self.dist_params.keys()
+        pred = dict(zip(names, dist_params_split))
 
-        # what?!? why is sigmoid(0) = 0?
+        # I don't think the special case for `x == 0` should be necessary
```

Comment:
Bryan agrees that this probably isn't necessary. I'm wary of making the change though without more reliable tests